pay/terms.go

```diff
@@ -92,6 +92,12 @@ func (t *Terms) Normalize() {
 	if t == nil {
 		return
 	}
+
+	if t.Detail != "" && t.Notes == "" {
+		t.Notes = t.Detail
+		t.Detail = ""
+	}
```

Comment:
Perhaps if both fields are present the Detail should be added after any existing notes? not sure about this one.